benchmarks/models.py

```diff
@@ -19,6 +19,10 @@
 
 class ManifestReduced(models.Model):
     hash = models.CharField(primary_key=True, max_length=40)
+    created_at = models.DateTimeField(default=timezone.now)
```

Comment:
I deleted all reduced manifests that had no real manifests attached and the SQL issue was still there. I had to add 'null=True' in the model and in migration to avoid the error.